javascript/javascript1/week4/index.js

```diff
@@ -0,0 +1,141 @@
+//Voice assistant
+let personName = "";
+let todoList = [];
+
+function getReply(command) {
+	var options = {
+		// weekday: "long",
+		year: "numeric",
+		month: "long",
+		//day: "numeric",
+	};
+	var today = new Date();
+	const date = today.getDate();
+
+	const myFormattedDate =
+		date + ". of " + today.toLocaleDateString("en-US", options);
+
+	let text = command.toLowerCase();
+	let response = null;
+
+	if (
+		text.includes("hello") &&
+		text.includes("my") &&
+		text.includes("name") &&
+		personName === ""
+	) {
+		personName = text.split(" ").pop();
+		//console.log("name=" + personName);
+		response = `Nice to meet you ${personName}`;
+	} else if (text.includes("hello my name is") && personName != "") {
+		response = `${personName} Hi again!`;
+	} else if (text.includes("what") && text.includes("name")) {
+		response = `Your name is ${personName}`;
+	} else if (text.includes("add fishing to my todo")) {
+		todoList.push("fishing");
+		response = `${todoList[0]} added to your todo`;
+	} else if (text.includes("add singing in the shower to my todo")) {
+		todoList.push("singing in the shower");
+		response = `${todoList[1]} added to your todo`;
```

Comment:
These 2 else if does the same thing adding todo to an array. Try to write a single if condition to do it. 
https://en.wikipedia.org/wiki/Don%27t_repeat_yourself